resources/views/comment.blade.php

```diff
@@ -90,7 +92,7 @@ class="text-xs text-gray-300 ml-1"
         @endif
     </div>
 
-    @if ($comment->isComment() && $comment->canDelete())
+    @if ($comment->isComment() && auth()->user()?->can('delete', $comment))
```

Comment:
We should be using `Config::resolveAuthenticatedUser()->can` here instead of the `auth()->user()` directly